docs/development/guidelines.md

```diff
@@ -370,7 +429,43 @@ Examples
 
 ## Tests
 
-If a function contains more code than just the getting or setting of a value, automated test should be added to the [`tests`][tests-folder] folder. The file structure in the tests folder should mirror the file structure of the [`src`][src-folder] folder.
+If a function contains more code
+than just the getting or setting of a value,
+automated test should be added
+to the [`tests`][tests-folder] folder.
+The file structure in the tests folder
+should mirror the file structure
+of the [`src`][src-folder] folder.
+
+Names of test functions
+shall start with `test_should_`
+followed by a description
+of the expected behaviour,
+e.g. `test_should_add_column`.
+
+### Testcases
+
+Tests should be parametrized
+using `@pytest.mark.parametrize`,
+even if there is only a single testcase.
```

Comment:
Add reason: Easier to add new test cases in the future.

Reply:
Done.